exaudfclient/base/javacontainer/script_options/converter_legacy.cc

```diff
@@ -0,0 +1,26 @@
+#include "base/javacontainer/script_options/converter_legacy.h"
+#include "base/javacontainer/script_options/string_ops.h"
+#include <iostream>
+#include <sstream>
+
+namespace SWIGVMContainers {
+
+namespace JavaScriptOptions {
+
+ConverterLegacy::ConverterLegacy()
+: Converter()
+, m_jarPaths() {}
+
+void ConverterLegacy::convertExternalJar(const std::string& value) {
+    std::istringstream stream(value);
+    std::string jar;
+
+    while (std::getline(stream, jar, ':')) {
+        m_jarPaths.insert(jar); // If m_jarPaths is a Set or you make it a Set, uniqueness implicitly is taken care of
```

Comment:
```suggestion
        m_jarPaths.insert(jar);
```
The comment was intended rather as additional info for the reviewee

Reply:
Removed it